lib/pbench/common/logger.py

```diff
@@ -88,7 +88,7 @@ def format(self, record):
         record.message = record.getMessage()
         if self._fmt.find("{asctime}") >= 0:
             try:
-                record.asctime = datetime.utcfromtimestamp(record.asctime).isoformat()
+                record.asctime = datetime.utcfromtimestamp(record.created).isoformat()
```

Comment:
Can we make this change in a separate PR?  I am not sure about this one, and would like to consider this on its own.

Reply:
Sure; mostly I wanted to be sure it really fixed the problem in Jenkins, since that's behaving differently from my local environment. Since the tests were succeeding before despite the traceback, I guess it's safe to back that out for this PR.